forge-gui/res/cardsfolder/t/tromokratis.txt

```diff
@@ -3,5 +3,5 @@ ManaCost:5 U U
 Types:Legendary Creature Kraken
 PT:8/8
 S:Mode$ Continuous | Affected$ Card.Self+notattacking+notblocking | AddKeyword$ Hexproof | Description$ CARDNAME has hexproof unless it's attacking or blocking.
-K:CARDNAME can't be blocked unless all creatures defending player controls block it.
+S:Mode$ MinMaxBlocker | ValidCard$ Card.Self | Min$ Count$Valid Creature.DefenderCtrl | Secondary$ True | Description$ CARDNAME can't be blocked unless all creatures defending player controls block it. (If any creature that player controls doesn't block this creature, it can't be blocked.)
```

Comment:
@Northmoc 
somehow I don't think this can work in all cases, because DefenderCtrl needs it to be an declared attacker already
but this check is usually run preemptively...

Reply:
The check does run a whole bunch, too

Is there a case I should check?

Reply:
this call in `AiAttackController.doAssault()` is done before attackers get declared:
![image](https://github.com/Card-Forge/forge/assets/8506892/5d3297f3-9fbe-42ae-9226-ebbf386d8e4c)

so maybe it miscalculates, though arguably it isn't that big of a deal

Reply:
This restores the old code without going back to the keyword... and now Player defender is actually used in applyMinMaxBlockerAbility, so that's nice

Seems like it should be no worse than before now?

Reply:
seems good :)